README.md

```diff
@@ -1,5 +1,20 @@
 # openworm_docs
 
-Documentation for OpenWorm, hosted at http://docs.openworm.org.  Information on contributing available at:
+Documentation for OpenWorm, hosted at <http://docs.openworm.org>.
 
-http://docs.openworm.org/en/latest/community#contributing-to-the-openworm-documentation
+[![CircleCI][CI-badge]][CI]
+
+Simply modifying the markdown files in this repository should automatically
```

Comment:
I created dev_openworm_docs because I didn't (and still don't) have maintainer-level access to https://readthedocs.org/projects/openworm/

@slarson or @travs - could you give @casperdcl and myself access?